test/behaviour/connection/database/BUILD

```diff
@@ -30,6 +30,7 @@ java_library(
     visibility = ["//visibility:public"],
     deps = [
         # Internal Package Dependencies
+        "//:client-java",
```

Comment:
Why do we need to depend on `//:client-java` now? Is it because we now return `Database` rather than `String`? Or are there other reasons?

Reply:
Yeah I think that's literally it haha, we need to import `Database`